frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -113,13 +113,5 @@ export const Marketplace = () => {
 Marketplace.displayName = 'Marketplace';
 
 export const MarketplacePage = () => {
-  return <React.Fragment>
-    <Helmet>
-      <title>Kubernetes Marketplace</title>
-    </Helmet>
-    <div className="co-catalog">
-      <PageHeading title="Kubernetes Marketplace" />
-      <Marketplace />
-    </div>
-  </React.Fragment>;
+  return <Marketplace />
```

Comment:
I believe you can simply rename `Marketplace` to `MarketplacePage` and remove this `MarketplacePage` component. Or even better, converge all three (Marketplace, MarketplacePage, and MarketplaceListPage) into one MarketplacePage component.